woocommerce/woocommerce
# OBW: Add wc-admin to recommended page

### All Submissions:






### Changes proposed in this Pull Request:

Add wc-admin to the Onboarding Wizard's Recommended page. If selected, WooCommerce Admin will be automatically installed and activated.

Changes here also reflect changes in copy, p90Yrv-16T-p2

### How to test the changes in this Pull Request:

1. On a fresh install (or just delete wc-admin on a test site).
2. run `grunt css` to see the css changes.
2. Go through the OBW and select WooCommerce Admin on the recommended page.
3. Complete the OBW and see wc-admin installed and active.

<img width="740" alt="Screen Shot 2019-07-03 at 8 43 26 PM" src="https://user-images.githubusercontent.com/1922453/60580760-c203be00-9dd9-11e9-9f7b-fec6b508573a.png">

### Changelog entry

Enhancement - Add WooCommerce Admin to recommended plugins in the Onboarding Wizard.
